test/libsolidity/util/ContractABIUtils.cpp

```diff
@@ -255,6 +260,21 @@ bool ContractABIUtils::appendTypesFromName(
 	return true;
 }
 
+optional<ABIType> ContractABIUtils::isFixedPoint(string const& type)
```

Comment:
Why don't you put this at the same place where all the other `is...` functions are? Does this need to be a static member of the class?

Reply:
Thats true!